src/pymovements/events/ivt.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import annotations
+
+from typing import Optional
+import numpy as np
+
+from pymovements.transforms import vnorm
+from pymovements.events import Fixation
+
+
+def ivt(
+        x: list[list[float]] | np.ndarray,
+        v: Optional[list[list[float]] | np.ndarray],
+        t: float
+) -> list[Fixation]:
+    """
+    Identification of fixations based on velocity-threshold
+
+    Parameters
+    ----------
+    x: array-like
+        Continuous 2D position time series.
+    v: array-like
+        Corresponding continuous 2D velocity time series.
+    t: float
+        Velocity threshold.
+
+    Returns
+    -------
+    fixations: array
+        List of fixations
+
+    """
+    x = np.array(x)
+    v = np.array(v)
+
+    # Check if threshold is greater 0
+    if not t > 0:
+        raise ValueError(
+            'velocity threshold must be greater than 0'
+        )
+
+    # make sure x and v have shape (n, 2)
+    if x.ndim != 2 and x.shape[1] != 2:
+        raise ValueError(
+            'x needs to have shape (n, 2)'
+        )
+
+    if x.ndim != 2 and x.shape[1] != 2:
+        raise ValueError(
+            'v needs to have shape (n, 2)'
+        )
+
+    # Check matching shape for x and v
+    if not x.shape == v.shape:
+        raise ValueError(
+            "shape x {} doesn't match shape v {}".format(x.shape, v.shape)
+        )
+
+    total_velocity = vnorm(v)
+
+    # Map velocities lower than threshold to 1 and greater equals to 0
+    fix_map = np.array(list(map(lambda point: 1 if point < t else 0, total_velocity)))
```

Comment:
I think this is a reasonable usage of list map lambda, but in a lot of cases list map lambda is not easily readable for users outside of the bubble/package, maybe if there is a neat way to reformulate it do it -- not a must

Reply:
or maybe explicit mention that `1` means fixation; 0 saccade